### PR TITLE
Fix for feedback

### PR DESCRIPTION
**Pull request by omorilewa**
#### What does this PR do?
- initial fix for feedback
- Update Screen would be used to add links to update the apk. Idea has not been concluded

### CODE REVIEW

File: src/components/Update.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { View } from 'react-native';
+import { StyledText as Text, SubHeader } from '.';
+
+const Update = () => (
+  <View >
+    <SubHeader pageHeaderText="UPDATE" />
+    <Text>Hello</Text>
+  </View>
```
Comment (ghost) [inline src/components/Update.js:9]: What is this component for ma?

Comment (omorilewa) [reply src/components/Update.js:9]: Lol. I just updated the description of the PR now.

Comment (ghost) [reply src/components/Update.js:9]: Awesome. Seen!

**Review by ghost — APPROVED** [review]
LGTM.

File: App.js [hunk 2]
```diff
@@ -18,16 +18,41 @@ import { Sentry } from 'react-native-sentry';
...
 Sentry.config('https://c9d63a6d35ca4c2bac30a2b979f5d68e:a42fa9660e7f472caa041e7d9db8ee49@sentry.io/1206436').install();
 
+const defaultGetStateForAction = PrimaryNavigator.router.getStateForAction;
+
 class App extends Component {
 
   componentDidMount() {
     // After having done stuff (such as async tasks) hide the splash screen
     SplashScreen.hide();
+    if (Platform.OS === 'android') {
```
Comment (omorilewa) [inline App.js:34]: @oreoluwade This bit prevent you from going back on the main screen.

**Pull request merged**